mcp/src/main/java/io/modelcontextprotocol/spec/McpClientSession.java

```diff
@@ -229,27 +229,27 @@ private String generateRequestId() {
 	public <T> Mono<T> sendRequest(String method, Object requestParams, TypeReference<T> typeRef) {
 		String requestId = this.generateRequestId();
 
-		return Mono.deferContextual(ctx -> Mono.<McpSchema.JSONRPCResponse>create(sink -> {
+		return Mono.deferContextual(ctx -> Mono.<McpSchema.JSONRPCResponse>create(responseSink -> {
```

Comment:
`responseSink` vs `resultSink` still feel quite similar. I have a different proposal.
Sink from `Mono.create` shall be called `pendingResponseSink`.
Sink for `handle` operator shall be called `deliveredResponseSink`.
WDYT?

Reply:
I like it